composer/loggers/neptune_logger.py

```diff
@@ -232,7 +238,11 @@ def upload_file(
             return
 
         del state  # unused
-        self.base_handler[remote_file_name].upload(str(file_path))
+
+        from neptune.types import File
+
+        with open(str(file_path), 'rb') as fp:
+            self.base_handler[remote_file_name].upload(File.from_content(fp.read(), extension=file_path.suffix))
```

Comment:
Won't this still fail if the file has been deleted before we have uploaded it?

Reply:
We do a similar thing in `neptune-detectron2`, though we use `from_stream`. Not sure if it's a big difference here, but I might be mistaken

Reply:
This should be fine.. I was more concerned that the file might have been removed from `file_path` before we can `open()` it, but that should never be the case ideally